boot/seal.go

```diff
@@ -46,6 +50,7 @@ func bootChainsFileUnder(rootdir string) string {
 
 // sealKeyToModeenv seals the supplied key to the parameters specified
 // in modeenv.
+// It assumes to be invoked in install mode.
```

Comment:
thanks for this comment

Reply:
Perhaps we should thing about a more fitting name for this function at some point.